Periodic_4_hyperbolic_triangulation_2/test/Periodic_4_hyperbolic_triangulation_2/test_p4ht2_complex.cpp

```diff
@@ -0,0 +1,57 @@
+#include <CGAL/CORE_Expr.h>
```

Comment:
Filename looks strange with respect to what this tests.

Reply:
Modified filename to `test_p4ht2_exact_complex_numbers.cpp`